src/api_server/src/request/drive.rs

```diff
@@ -112,28 +43,32 @@ pub fn parse_patch_drive(body: &Body, id_from_path: Option<&&str>) -> Result<Par
         return Err(Error::EmptyID);
     };
 
-    let patch_drive_payload = PatchDrivePayload {
-        fields: serde_json::from_slice(body.raw()).map_err(|e| {
-            METRICS.patch_api_requests.drive_fails.inc();
+    let block_device_update_cfg: BlockDeviceUpdateConfig =
+        serde_json::from_slice::<BlockDeviceUpdateConfig>(body.raw()).map_err(|e| {
+            METRICS.put_api_requests.drive_fails.inc();
```

Comment:
Shouldn't this be `METRICS.patch_api_fails.drive_fails.in()`?

Reply:
Fixed.